### ISSUE TITLE
Taxii server data published in wrong format

### ISSUE THREAD

**Issue by kknagesh** [MISP/MISP#4196]
Hello Team, We have MISP Application running and we are loading the data from the taxii server installed on the same server. 

The data is being published to the MISP but not in the correct format. 
For Example => The Title says the "Card is comprimised" in the source (when i check the data in the taxii database), however when it is published to MISP, it says STIX Import.

Can you please let me know how this can be fixed

**Comment by adulau** [comment]
Do you have the original STIX file which you tried to import?

**Comment by kknagesh** [comment]
Nope.. The taxii server is getting the data from the external source and loading it to the MISP.

**Comment by kknagesh** [comment]
I Just tried a content-block saved in xml file and then import that xml file. I am just getting the observables part loaded.. however the Indicators are not loaded in the MISP

I have Indicators , TTP, Incidents after the observables which are not loaded

**Comment by adulau** [comment]
Can you share the STIX files with us?

**Comment by kknagesh** [comment]
[vendor.txt](https://github.com/MISP/MISP/files/2900236/vendor.txt)
this is the file which we are uploading and getting the above observation.
please change the extension to xml

**Comment by kknagesh** [comment]
The issue what we are noticing right now is we are successfully uploading the STIX 1.2 to the MISP using a Taxii-Server. However any feeds which are less than STIX1.2 (says STIX 1.1 , 1.1.1) we are not able to upload.
